src/css/popup-new.css

```diff
@@ -295,3 +299,92 @@ sign-up-panel::after {
   background-color: var(--colorViolet05);
   border-radius: var(--borderRadiusSm);
 }
+
+/* Report Panel */
+
+.report-issue-content {
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacingMd);
+  padding: var(--spacingMd);
+  border: 0 none;
+  font-size: 14px;
+}
+
+.report-section {
+  margin-bottom: var(--spacingMd);
+  display: flex;
+  gap: var(--spacingMd);
+  flex-direction: column;
+}
+
+.report-label::after {
+  content: ":";
+}
+
+.report-section ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.report-section li {
+  display: flex;
+  align-items: flex-start;
+  margin-bottom: var(--spacingMd);
+}
+
+.report-section input[type=checkbox]{
+  margin: 0;
+  margin-top: 2px; /* Custom Alignment Issue with Checkbox and Label */
```

Comment:
```suggestion
```

Try instead setting `align-items: center` on `.report-section li`.

Reply:
I tried that but no dice. Because a few of the checkboxes break over to two lines, it's too centered? 

<img width="393" alt="image" src="https://user-images.githubusercontent.com/2692333/214335522-519a6785-e0b3-466a-bd97-d185ee28bc5b.png">

Reply:
Oh, that looks fine to me, and is more robust to zooming and different checkbox layouts on different browsers/operating systems, but I see what you mean. Then maybe at least mention the multi-line issue in the comment :)

One other approach you could try is to set an explicit line height and checkbox height, which might be somewhat more robust to different sizes, but that's probably not worth the effort.

Reply:
Filed this to capture this conversation and come back in clean up. https://github.com/mozilla/fx-private-relay-add-on/issues/460